client/wfnews-war/src/main/angular/src/app/components/public-incident-page/incident-info-header/incident-header-panel.component.ts

```diff
@@ -104,4 +104,16 @@ export class IncidentHeaderPanel implements AfterViewInit {
   removeFromWatchlist () {
     this.watchlistService.removeFromWatchlist(this.incident.incidentNumberLabel)
   }
+
+  displaySizeType (incidentSizeDetail: string) {
+    if (incidentSizeDetail && incidentSizeDetail.includes('estimated')) {
+      return '(Estimated)'
```

Comment:
Would probably make sense to move the brackets out into the HTML rather than part of the output here